node/core/dispute-coordinator/src/initialized.rs

```diff
@@ -269,8 +269,13 @@ impl Initialized {
 		update: ActiveLeavesUpdate,
 		now: u64,
 	) -> Result<()> {
-		let on_chain_votes =
+		let scraped_updates =
 			self.scraper.process_active_leaves_update(ctx.sender(), &update).await?;
+		log_error(
```

Comment:
I wonder can't we just do:
```
self.participation
				.bump_to_priority_for_candidates(ctx, &scraped_updates.included_receipts)
				.await?;
```

Why we need `log_error`?

Reply:
There are non-fatal errors that we can expect somewhat regularly in `bump_to_priority_for_candidates`. So `log_error` is there to process any non-fatal errors without returning. For instance we'd sometimes try to bump a best effort queue entry to priority, find that priority doesn't have room, and exit from the entire dispute coordinator `process_active_leaves_update` call. Also, log_error() is used in the other place where we try to add to the priority queue. So this is consistent.

Reply:
Agreed. Good thinking, we should not fail the whole function just because the priority queue is full.